README.md

```diff
@@ -180,6 +180,18 @@ GenericEwonDevice is a Thing template that applies to all Ewon device Things cre
 
 5. *WriteStringTagGenericService*: A generic service used by the GenericEwonDeviceTalk2M, GenericEwonDeviceDirect, and GenericEwonDeviceDirectStandalone Thing Templates for writing a value to a string tag on the Ewon Device.
 
+6. *ReadAllTagsLiveGenericService*: A generic service used by the GenericEwonDeviceTalk2M, GenericEwonDeviceDirect, and GenericEwonDeviceDirectStandalone Thing Templates for reading all live tag values on the Ewon Device.
```

Comment:
This is a suggestion. "current tag value" can be more descriptive than "live tag value" and these could be changed.

Reply:
The name "live" was picked because it more clearly differs these M2Web tag reads from the current tag value reported by the Ewon device from historical data.